### DISCUSSION TITLE
Loading local distribution of WaveSurfer

### DISCUSSION THREAD

**Discussion by sojastar** [katspaugh/wavesurfer.js#2767] [Q&A] [answered]
Hello,

I am working on an art project that involves playing audio from a webserver that is not connected to the internet. Visitors connect to a local wifi network then access a webpage where the audio content is made available to them. I would like to use wavesurfer.js to display a waveform of each audio file. Since the server is off the internet, I can't use `<script src="https://unpkg.com/wavesurfer.js"></script>` to import wavesurfer.js. Also, the site/webapp is developed with Ruby Sinatra (because I'm old) and not nodejs, so I can't use `var WaveSurfer = require("wavesurfer.js");`.

Any idea on how I could have a local distribution of wavesurfer.js imported either in the html or javacript of the application would be greatly apreciated.

Thank you very much!

**Comment by katspaugh** [comment, marked as answer]
You can open https://unpkg.com/wavesurfer.js and copy and paste the code into a local file. Then insert via `<script src="./local-file.js"></script>`

**Comment by sojastar** [comment]
That works perfectly! Thank you!
